docs/howto/account-billing/change-credit-card.md

```diff
@@ -0,0 +1,33 @@
+# Changing to new credit card
+
+You can only change your credit card when you _pay a new invoice_ via [{{gui}}](https://{{gui_domain}})
+
+## Prerequisites
+
+You must be logged in at your [Cleura 
+Cloud](https://{{gui_domain}}) account to switch credit card.
```

Comment:
Say "switch credit cards" or "switch to a new credit card", but not "switch credit card".